src/citrine/resources/file_link.py

```diff
@@ -236,10 +280,12 @@ def _make_upload_request(self, file_path: str, dest_name: str):
         try:
 
             uploader.region_name = upload_request['s3_region']
-            uploader.aws_access_key_id = upload_request['temporary_credentials']['access_key_id']
+            uploader.aws_access_key_id = upload_request['temporary_credentials'][
+                'access_key_id']
```

Comment:
linting clean-up?  There are a couple changes like this that don't really improve legibility.

Reply:
I assume that's my IDE doing its thing. I'll see about backing out some of the useless ones.